Quran/HighlightingView.swift

```diff
@@ -8,12 +8,22 @@
 
 import UIKit
 
+@objc protocol HighlightingViewDelegate {
```

Comment:
Is there a specific reason to have `@objc` protocol? I think that protocol can just be constrained to be a class protocol `protocol HighlightingViewDelegate: class``.